sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -812,6 +862,43 @@ public async Task StartRecognizeCustomFormsWithLabels(bool useStream, bool inclu
             Assert.AreEqual("948284", form.Fields[name].ValueData.Text);
         }
 
+        [Test]
+        [TestCase(true)]
+        [TestCase(false)]
+        public async Task StartRecognizeCustomFormsWithLabelsAndSelectionMarks(bool includeFieldElements)
+        {
+            var client = CreateFormRecognizerClient();
+            var options = new RecognizeCustomFormsOptions { IncludeFieldElements = includeFieldElements };
+            RecognizeCustomFormsOperation operation;
+
+            await using var trainedModel = await CreateDisposableTrainedModelAsync(useTrainingLabels: true, ContainerType.SelectionMarks);
+
+            using var stream = FormRecognizerTestEnvironment.CreateStream(TestFile.FormSelectionMarks);
+            using (Recording.DisableRequestBodyRecording())
+            {
+                operation = await client.StartRecognizeCustomFormsAsync(trainedModel.ModelId, stream, options);
+            }
+
+            await operation.WaitForCompletionAsync(PollingInterval);
+
+            Assert.IsTrue(operation.HasValue);
+            Assert.GreaterOrEqual(operation.Value.Count, 1);
+
+            RecognizedForm form = operation.Value.Single();
+
+            ValidateRecognizedForm(form, includeFieldElements: includeFieldElements,
+                expectedFirstPageNumber: 1, expectedLastPageNumber: 1);
+
+            // Testing that we shuffle things around correctly so checking only once per property.
+            Assert.IsNotEmpty(form.FormType);
+            Assert.AreEqual(1, form.Pages.Count);
+            Assert.IsNotNull(form.Fields);
+            var name = "Checkbox_Paramedic Subscription_Amount Enclosed";
+            Assert.IsNotNull(form.Fields[name]);
+            Assert.AreEqual(FieldValueType.SelectionMark, form.Fields[name].Value.ValueType);
+            Assert.AreEqual("selected", form.Fields[name].ValueData.Text);
```

Comment:
Thank u for pointing this out. I looked and because Text comes from the service we have it lowercase. I made a modification in `FieldValue_internal` so that we parse text to a `FormSelectionMarkState` and then `ToString` so we are consistent